src/components/TerrainTypes.cs

```diff
@@ -61,23 +61,10 @@ public int GetMovementCost()
 
 	public override string ToString()
 	{
-		string s = "";
-
-		for (int i = 0; i < List.Count; i++)
-		{
-			s += List[i].ToString();
-
-			if (i < List.Count - 1)
-			{
-				s += ", ";
-			}
-			
-		}
+		string s = string.Join(", ", List);
 		foreach (var type in List)
```

Comment:
It was me who forgot left it there accidentally but since this is a cleanup PR can you remove that empty for loop there? :)